WolframLanguage.sublime-syntax

```diff
@@ -90,14 +106,19 @@ contexts:
         pop: true
 
   literals:
-    - match: (?:\d+\.?|\.\d)\d*\`{0,2}
+    # numbers, with base, scientific notation, precision or accuracy
+    - match: ([0-9]+\^\^)?{{number}}(\`\`?({{number}})?)?(\*\^{{number}})?
       scope: constant.numeric.wolfram
+
+    # built-in symbols
     - match: ((?:System`)?{{built_in_numeric_constants}})
       scope: constant.numeric.wolfram
     - match: ((?:System`)?{{built_in_constants}})
       scope: constant.language.wolfram
     - match: ((?:System`)?{{built_in_options}})
-      scope: variable.function.wolfram
+      scope: variable.parameter.wolfram
```

Comment:
I know this is conceptually wrong but I think `function` was there to have the same colouring for all System symbols.

Reply:
But built-in-constants are displayed as "constants".